cmd/volume.go

```diff
@@ -82,21 +91,23 @@ func VolumeList(cmd *cobra.Command, args []string) error {
 }
 
 func VolumeResize(cmd *cobra.Command, args []string) error {
-	var name string
+	var name = args[0]
+	var sizeStr *string
 
+	if len(args) > 1 {
+		sizeStr = &args[1]
+	}
 	if name == "" {
-		ui.Errorf("Invalid volume name")
-		os.Exit(1)
+		ui.Fatal("Invalid volume name", nil)
```

Comment:
Fatal currently doesn't always exit with `os.Status(1)`. Cloud you change that so it does?

Reply:
done